go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/kubernetes-csi/external-snapshotter/client/v4 v4.2.0
 	github.com/onsi/ginkgo/v2 v2.19.0
 	github.com/onsi/gomega v1.33.1
-	github.com/openshift/api v0.0.0-20230213134911-7ba313770556 // release-4.12
+	github.com/openshift/api v0.0.0-20240524162738-d899f8877d22 // release-4.12
```

Comment:
if you had to bump this, please update comments accordingly if this commit is from a certain release-4.x branch.

Reply:
https://github.com/openshift/api/commit/d899f8877d22
This is coming from master..

Reply:
matches release-4.19 for the time being.. if that differ in the future, or when we do oadp-1.5 branching, we should use the stable branches.